README.md

```diff
@@ -249,6 +251,31 @@ Kubernetes secrets:
 $ kubectl create secret generic spid-cert --from-file=./cert.pem --from-file=./key.pem
 ```
 
+### SAML (SPID) certs with ENV vars
+
+The certificate and the secret key could be provided to the backend with two ENV variables:
+- `SAML_KEY`
+- `SAML_CERT`
+
+if these variables are set `cert.pem` and `key.pem` are ignored.
+
+### SPID Identity Providers Metadata
+
+When backend starts, SPID login is configured with Identity Providers metadata fetched from remote URL. URLs are provided by the following Environment variables: `CIE_METADATA_URL` (CIE login) `IDP_METADATA_URL` (SPID login) and `SPID_TESTENV_URL` (SPID test environment). Before that the backend become ready a time consuming operation must be completed (fetching remote metadata).
+
+The default values are explained into the following table
+
+| ENV                                    | stage                                                                             | production                                                       |
+|----------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| `IDP_METADATA_URL`                       | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml                  | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml |
+| `CIE_METADATA_URL`                       | https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth                    | https://idserver.servizicie.interno.gov.it/idp/shibboleth        |
+| `SPID_TESTENV_URL`                       | https://spid-testenv.dev.io.italia.it                                             | -                                                                |
```

Comment:
it looks like we won't have a staging environment deployed (only production)
so these settings here should be the ones relative to local development setup (through docker)

Reply:
can you add the settings to setup spid validator integration ? (local dev only, to let developers login using docker-compose and verify spid validator constraints)

Reply:
Fixed